src/pages/UserAccount/User.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './SignUp.scss';
+
+function User({ userData, isSelectLogin }) {
+  const { title, text, url, button } = userData;
```

Comment:
```suggestion
function User({ userData: { title, text, url, button }, isSelectLogin }) {
```

구조분해할당을 위와 같이 인자에서 한번 더 진행할 수 있습니다!